src/components/ledger/LedgerConnector.tsx

```diff
@@ -81,6 +82,8 @@ export function LedgerConnector({
   const [addresses, setAddresses] = useState<AddressType[]>([]);
   const [hasPublicKeys, setHasPublicKeys] = useState(false);
   const [dropdownDisabled, setDropdownDisabled] = useState(true);
+  const lastAccountIndexWithBalance = useRef(0);
```

Comment:
Why `useRef()` instead of `useState()`?

Reply:
We don't display that value just use it in calculation therefor rerendering is not necessary. It saves the cost of rerendering and other side effects of that.